src/index.js

```diff
@@ -29,18 +29,19 @@ videojs.use('video/mux', (player) => {
 
   return {
     setSource({ src }, next) {
+      const { customDomain:domain = 'mux.com', timelineHoverPreviews } = player.options();
```

Comment:
`customDomain: domain` you're missing a space, but I would have thought our linter would catch that

Reply:
Ah yea I can add that.  It seems we don't have linting added to this project.  We have prettier, but I couldn't find where we actually invoke the thing.